example/tailable_cursor.dart

```diff
@@ -2,22 +2,34 @@ import 'package:mongo_dart/mongo_dart.dart';
 import 'dart:async';
 
 //////// I guess tailable cursor does not work in that example
+/// NJ: it works for me provide the collection is not empty...
 main() async {
-  var db = new Db("mongodb://127.0.0.1/local");
+  var db = new Db("mongodb://127.0.0.1/test");
 
   await db.open();
-    var oplog = new DbCollection(db, "log");
-    Cursor cursor = oplog.createCursor()
-        ..tailable  = true
-        ..timeout   = false
-        ..awaitData = false;
-   while (true) {
-      var doc = await cursor.nextObject();
-      if (doc == null) {
-        await new Future.delayed(new Duration(seconds: 1), () => null);
-      } else {
-        print(doc);
-      }
-   }
-
-}
\ No newline at end of file
+  var i = 0;
+  await db.collection("log").insert({"index": i});
+  new Timer.periodic(new Duration(seconds: 10), (Timer t) async {
```

Comment:
@Pacane : in this example, the 2 actions (inserting/fetching) occurs in parallel. 
Using a for loop would make the 2 operation serial (all instert, then all fetch) which is  not what i wanted to show.